docs/resources/settings.md

```diff
@@ -0,0 +1,30 @@
+---
+subcategory: "Settings"
+---
+
+# databricks_namespace_settings Resource
+
+The `databricks_namespace_settings` resource allows you to set the settings for the default namespace.
```

Comment:
Should we explain what `default namespace` is? At least we need to have a link to the docs.

Reply:
There's no public documentation in Prod yet for UC as default.